modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java

```diff
@@ -241,6 +244,38 @@ public DDMFormInstanceRecord deleteFormInstanceRecord(
 			DDMFormInstanceRecord ddmFormInstanceRecord)
 		throws PortalException {
 
+		return deleteFormInstanceRecord(ddmFormInstanceRecord, true);
+	}
+
+	@Indexable(type = IndexableType.DELETE)
+	@Override
+	@SystemEvent(
+		action = SystemEventConstants.ACTION_SKIP,
+		type = SystemEventConstants.TYPE_DELETE
+	)
+	public DDMFormInstanceRecord deleteFormInstanceRecord(
```

Comment:
Since the previous method calls this new one, you can remove the annotations of the prior method (leave the override). Remember to run the buildService again.

```
	@Override
	public DDMFormInstanceRecord deleteFormInstanceRecord(DDMFormInstanceRecord ddmFormInstanceRecord) throws PortalException {
		return deleteFormInstanceRecord(ddmFormInstanceRecord, true);
	}
```